### ISSUE TITLE
Cannot read properties of undefined (reading '1')

### ISSUE THREAD

**Issue by ryuji-orca** [mrcaidev/tailwindcss-radix-colors#12]
I got an error in the build of Next.js when I upgraded the dependency version👀

```
Failed to compile.

./node_modules/.pnpm/@radix-ui+themes@2.0.3_@types+react-dom@18.2.18_@types+react@18.2.48_react-dom@18.2.0_react@18.2.0/node_modules/@radix-ui/themes/styles.css.webpack[javascript/auto]!=!./node_modules/.pnpm/next@14.1.0_@babel+core@7.23.7_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[11].oneOf[12].use[2]!./node_modules/.pnpm/next@14.1.0_@babel+core@7.23.7_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[11].oneOf[12].use[3]!./node_modules/.pnpm/@radix-ui+themes@2.0.3_@types+react-dom@18.2.18_@types+react@18.2.48_react-dom@18.2.0_react@18.2.0/node_modules/@radix-ui/themes/styles.css

TypeError: Cannot read properties of undefined (reading '1')
    at p1 (/Users/project/node_modules/.pnpm/tailwindcss-radix-colors@1.1.0_tailwindcss@3.4.1/node_modules/tailwindcss-radix-colors/dist/index.cjs:1:105851)
```

Is it because of tailwindcss-radix-colors?

Below are the versions of the packages that may be involved.
```
    "@headlessui/tailwindcss": "^0.2.0",
    "@radix-ui/react-label": "^2.0.2",
    "@radix-ui/react-navigation-menu": "^1.1.4",
    "@radix-ui/react-radio-group": "^1.1.3",
    "@radix-ui/react-select": "^2.0.0",
    "@radix-ui/react-slot": "^1.0.2",
    "@radix-ui/react-toast": "^1.1.5",
    "@radix-ui/themes": "^2.0.3",
    "next": "^14.1.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-hook-form": "^7.49.3",
    "tailwind-merge": "^2.2.1",
    "tailwindcss-radix-colors": "^1.1.0",

// devDependencies
    "@types/node": "^20.7.2",
    "@types/react": "^18.2.22",
    "@types/react-dom": "^18.2.7",
    "eslint": "^8.56.0",
    "eslint-plugin-tailwindcss": "^3.14.0",
    "tailwindcss": "^3.4.0",
    "typescript": "^5.2.2"
```

**Comment by bearkfear** [comment]
same here

**Comment by mrcaidev** [comment]
Can't reproduce with the same dependencies. Any repository or sandbox available to be cloned so that I can test it?

And try update this plugin to `1.2.0` to see if the problem still exists.

**Comment by ryuji-orca** [comment]
@mrcaidev 
I still get an error even with `1.2.0`.
Since this is a project that cannot be made public, I will create another one to share on Saturday or Sunday 🙏.

**Comment by yxshv** [comment]
> Can't reproduce with the same dependencies. Any repository or sandbox available to be cloned so that I can test it?
> 
> And try update this plugin to `1.2.0` to see if the problem still exists.

I am having the same error using nextjs, tried downgrading... same error

**Comment by emi0x7d1** [comment]
I had to delete the `colors` entry in my theme.

**Comment by yxshv** [comment]
> I had to delete the `colors` entry in my theme.

Well that is a problem

**Comment by mrcaidev** [comment]
Sorry for the late reply, but this issue should be fixed in `v1.3.0`. Try it out and let me know if there is any problem.
